invenio_records_resources/resources/errors.py

```diff
@@ -116,56 +118,59 @@ class ErrorHandlersMixin:
         QuerystringValidationError: create_error_handler(
             HTTPJSONException(
                 code=400,
-                description="Invalid querystring parameters.",
+                description=_("Invalid querystring parameters."),
             )
         ),
         PermissionDeniedError: create_error_handler(
             HTTPJSONException(
                 code=403,
-                description="Permission denied.",
+                description=_("Permission denied."),
             )
         ),
         RecordPermissionDeniedError: create_error_handler(
             HTTPJSONException(
                 code=403,
-                description="Permission denied.",
+                description=_("Permission denied."),
             )
         ),
         PIDDeletedError: create_error_handler(
             HTTPJSONException(
                 code=410,
-                description="The record has been deleted.",
+                description=_("The record has been deleted."),
             )
         ),
         PIDAlreadyExists: create_error_handler(
             HTTPJSONException(
                 code=400,
-                description="The persistent identifier is already registered.",
+                description=_("The persistent identifier is already registered."),
             )
         ),
         PIDDoesNotExistError: create_error_handler(
             HTTPJSONException(
                 code=404,
-                description="The persistent identifier does not exist.",
+                description=_("The persistent identifier does not exist."),
             )
         ),
         PIDUnregistered: create_error_handler(
             HTTPJSONException(
                 code=404,
-                description="The persistent identifier is not registered.",
+                description=_("The persistent identifier is not registered."),
             )
         ),
         PIDRedirectedError: create_pid_redirected_error_handler(),
         NoResultFound: create_error_handler(
             HTTPJSONException(
                 code=404,
-                description="Not found.",
+                description=_("Not found."),
             )
         ),
         FacetNotFoundError: create_error_handler(
             lambda e: HTTPJSONException(
                 code=404,
-                description=f"Facet {e.vocabulary_id} not found.",
+                # using gettext here in order to be able to call the format method
+                description=gettext(
```

Comment:
i don't understand why only gettext could take parameters. [lazy_gettext](https://github.com/python-babel/flask-babel/blob/master/flask_babel/__init__.py#L677)  also takes `**variables`.

Reply:
right, will fix it